src/lib.rs

```diff
@@ -434,61 +501,61 @@ make_color! {
     ///Linear luminance.
     Luma {
         ///Linear luminance.
-        y(luma: T)[alpha: T] => new;
+        y(luma: T)[alpha: T] => with_wp;
```

Comment:
That's odd. It shouldn't complain about traits. Must be something in the way the `impl` is done. It should say `impl<T: Float> Color<D65, T>` if `new` is used.